tests/run-macros/gestalt-type-toolbox-reflect/Macro_1.scala

```diff
@@ -58,7 +58,7 @@ object TypeToolbox {
   inline def methodIn[T](inline mem: String): Seq[String] = ${methodInImpl('[T], 'mem)}
   private def methodInImpl(t: Type[_], mem: Expr[String])(using qctx: QuoteContext) : Expr[Seq[String]] = {
     import qctx.tasty._
-    Expr(t.unseal.symbol.classMethod(mem.value).map(_.name))
+    Expr(t.unseal.symbol.classMethod(mem.unlifted).map(_.name))
```

Comment:
I'm wondering whether it's better to remove `unlifted` and only keep `unlift`? As it seems to me in most cases, the usage of `.unlifted` in meta-programming is just wrong, it's inviting errors.

Reply:
Most of the cases in the tests use `unlifted` because I am not bothered to handle the failure myself in the test cases. Most should use `unlift`.

Reply:
Additionally `unlift` does what `Unlliftable` provides (i.e. return an `Option`)

Reply:
The only alternative would be to remove `unlifted` and use `unlift.getOrElse(ctx.throwError("..."))` everywhere.

Reply:
Or rename it to be `.unliftOrError`?

Reply:
Yes, that is what we need

Reply:
@liufengyun I renamed it